packages/patternfly-4/react-core/src/components/Page/PageSection.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Page/page';
+import { css } from '@patternfly/react-styles';
+
+export enum PageSectionVariants {
+  default = 'default',
+  light = 'light',
+  dark = 'dark',
+  darker = 'darker'
+};
+
+export interface PageSectionProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+  className?: string;
+  variant?: 'default' | 'light' | 'dark' | 'darker'
```

Comment:
Add `| PageSectionVariants;` for people using the old API.

Reply:
fixed